webapp/components/Editor/nodes/Hashtag.js

```diff
@@ -21,20 +21,30 @@ export default class Hashtag extends TipTapMention {
     return {
       ...super.schema,
       toDOM: node => {
+        // use a dummy domain because URL cannot handle relative urls
+        const url = new URL('/', 'https://human-connection.org')
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-16T08:40:04Z" title="Monday, September 16th 2019, 10:40:04 am +02:00">Sep 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -25,7 +25,7 @@ export default class Hashtag extends TipTapMention {
           'a',
           {
             class: this.options.mentionClass,
-            href: `/search/hashtag/${node.attrs.id}`,
+            href: `/search/hashtag/${encodeURI(node.attrs.id)}`,
```


Can't we use query parameters? :cry: 

I found out how to use [URL](https://nodejs.org/api/url.html) to encode/decode url params for relative URLs: Just use a dummy domain!

Example:

```js
const url = new URL('http://example.org')
url.searchParams.append('hashtag', 'ä  #')
url.search // returns ?hashtag=%C3%A4++%23
```